assets/scripts/aircraft.js

```diff
@@ -166,9 +170,51 @@ zlsa.atc.Conflict = Fiber.extend(function() {
         }
       }
       // Standard separation
+      /* 7110.65, section 5-5-7-a-1:
+         (a) Aircraft are on opposite/reciprocal
+         courses and you have observed that they have passed
+         each other; or aircraft are on same or crossing
+         courses/assigned radar vectors and one aircraft has
+         crossed the projected course of the other, and the
+         angular difference between their courses/assigned
+         radar vectors is at least 15 degrees.
+      */
       else {
-        conflict = (this.distance < 7.4); // 4nm
-        violation = (this.distance < 5.6); // 3nm
+        var offset = abs(angle_offset(this.aircraft[0].groundTrack,
+                                      this.aircraft[1].groundTrack));
+        //TODO: Opposing course doesn't seem to be handled correctly
+        // Check for courses differing by at least 15 degrees and within 4nm
+        if ((this.distance <= 7.4) && (offset > radians(15)))
```

Comment:
Should be **>=** 15 degrees here, seeing as 15 degree divergence *IS* okay, while 14.9 would be a sep loss, as you noted your comment.

Other than that, everything seems to work perfectly. Badass! :smile: